src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/ConsentOrderApprovedController.java

```diff
@@ -145,9 +143,8 @@
                 caseData = mapper.readValue(mapper.writeValueAsString(caseData), HashMap.class);
                 caseDetails.setData(caseData);
             }
-        } else {
-            caseData.put(UPLOAD_ORDER, caseData.get(GENERAL_ORDER_LATEST_DOCUMENT));
```

Comment:
whats the reason for removing this?

Reply:
I don't see it's required - there's logic in bulkprint methods added to send general order if there are no "contested case consented not approved" orders.